File: ValidAnagrm/VA.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):return False
        dic1 = []
        dic2 = []
        for n in s:
            dic1.append(n)
        for  n in t:
            dic2.append(n)
        dic1.sort()
        dic2.sort()
        for i in range(0,len(s)):
            if dic1[i]!=dic2[i]:return False
        return True

class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        return sorted(s) == sorted(t)

File: ValidAnagrm/VA.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()) return false;
        char[] a1 = s.toCharArray();
        char[] a2 = t.toCharArray();
        Arrays.sort(a1);
        Arrays.sort(a2);
        return Arrays.equals(a1,a2);
    }
}